lib/service/invoices.go

```diff
@@ -157,13 +157,14 @@ func createLnRpcSendRequest(invoice *models.Invoice) (*lnrpc.SendRequest, error)
 	if err != nil {
 		return nil, err
 	}
+	invoice.DestinationCustomRecords[KEYSEND_CUSTOM_RECORD] = preImage
 	return &lnrpc.SendRequest{
 		Dest:              destBytes,
 		Amt:               invoice.Amount,
 		PaymentHash:       pHash.Sum(nil),
 		FeeLimit:          &feeLimit,
 		DestFeatures:      []lnrpc.FeatureBit{lnrpc.FeatureBit_TLV_ONION_REQ},
-		DestCustomRecords: map[uint64][]byte{KEYSEND_CUSTOM_RECORD: preImage, TLV_WHATSAT_MESSAGE: []byte(invoice.Memo)},
```

Comment:
memo is now no longer added as default TLV_WHATSAT_MESSAGE?

Reply:
No, I've decided to leave this up to the client.
I've left the memo field to keep backwards compatibility, but if it is not added explicitly to the custom records by the client, it's only purpose is to be stored in the database as an internal memo field.

Reply:
ok. 👍